README.md

```diff
@@ -10,17 +10,19 @@ Docker image for hugo static page generator (https://gohugo.io)
 * `HUGO_DESTINATION` (Path where hugo will render the site. By default `/output`)
 * `HUGO_REFRESH_TIME` (in seconds, only applies if not watching, if not set, the container will build once and exit)
 * `HUGO_BASEURL`
+* `HUGO_NEW` (true for a new install)
 
 
 ## Executing
 
-    docker run --name "my-hugo" -P -v $(pwd):/src jojomi/hugo
+    docker run --name "my-hugo" -P -v $(mktemp):/myuser -v $(pwd):/src jojomi/hugo
 
 Or, more verbosely, and with a specified output mapping:
 
     docker run --name "my-hugo" --publish-all \
            --volume $(pwd):/src \
            --volume /tmp/hugo-build-output:/output \
+	   --volume $(mktemp):/myuser
```

Comment:
What does `/myuser` do?

Reply:
/myuser is a temporary file (indicated by mktemp) which is used to "detect" the UID/GID so the docker instance can assign the correct permissions when invoked as a user. Without this, there is no good way without boilerplating it somehow.

Reply:
Ah! Okay, so the file is created temporarily, and then mounted, providing a file inside the container from which the entryfile can extract the user ID. If that's the case, sweet!